Detectors/TPC/workflow/src/TrackAndClusterFilterSpec.cxx

```diff
@@ -43,6 +44,15 @@ class TrackAndClusterFilterDevice : public o2::framework::Task
     mTrackDump.outputFileName = ic.options().get<std::string>("output-file");
     mTrackDump.writeTracks = ic.options().get<bool>("write-tracks");
     mTrackDump.writeGlobal = ic.options().get<bool>("write-global-cluster-info");
+    mMindEdx = ic.options().get<double>("min-dedx");
+    mMaxdEdx = ic.options().get<double>("max-dedx");
```

Comment:
Perhaps the `dEdx` range can also be added to the `TrackCuts`.

Reply:
That is handy, I will do it.